### ISSUE TITLE
requiring react from test silently fails

### ISSUE THREAD

**Issue by neojski** [babel/babel-jest#35]
I just tried the demo code from react website:

```js
// __tests__/sum-test.js
jest.dontMock('../sum');

describe('sum', function() {
 it('adds 1 + 2 to equal 3', function() {
   var sum = require('../sum');
   expect(sum(1, 2)).toBe(3);
 });
});
```

When I add `require('react')` after `expect` above it fails like this:

```js
> npm test

> @ test /home/neo/demo
> jest


  == NOTICE: ==
    Node and io.js will be merging. http://bit.ly/1dOjLVK
    On August 1st, 2015, Jest v0.5.x will work only on io.js until the merger is complete
    Legacy v0.4.x for Node 0.10.x will still be available on
    on npm and from the "0.4.x" branch on GitHub.

Using Jest CLI v0.4.18
Waiting on 1 test...%
```

where that percent sign seems to indicate that there is no new line.

I think I might be missing some preprocessing stuff here but there's no error.

Oh, I copy-pasted `package.json` from README and installed react.

**Comment by neojski** [comment]
Just one more thing. When I did `browserify sum-test.js > sum-test-browserified.js`, removed `sum-test.js` and ran `npm test` it works.

**Comment by zertosh** [comment]
What version of node are you using?

**Comment by neojski** [comment]
I'm using node.js `v0.12.7`

**Comment by zertosh** [comment]
jest 0.4.x doesn't work with node v0.12.x. check out the notice in your description.

**Comment by neojski** [comment]
It doesn't seem to be clear at all that it was the reason. It said that `Legacy v0.4.x for Node 0.10.x will still be available` and I thought `0.12.7` was good enough.

I installed iojs and it looks like this problem went away. Thanks!
